config/HelixTaskWorkflowManagerConfig.properties

```diff
@@ -0,0 +1,18 @@
+#
+# Copyright (C) 2022 LinkedIn Corp. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use
+# this file except in compliance with the License. You may obtain a copy of the
+# License at  http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed
+# under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+# CONDITIONS OF ANY KIND, either express or implied.
+#
+
+zk.layout.file.path=../config/zkLayout.json
+cluster.name=<cluster name>
+workflow.name=<workflow name>
+stats.reports.to.aggregate=PARTITION_CLASS_REPORT,ACCOUNT_REPORT
+delete.specified.workflow=false
+recurrent.interval.in.minutes=10800
```

Comment:
Change this to 10, now that the aggregation task should be executed every 10 minutes

Reply:
done but this is just sample config